GAE/src/org/waterforpeople/mapping/app/web/rest/QuestionGroupRestService.java

```diff
@@ -141,6 +141,17 @@
                 }
             }
         }
+
+        // in this case, we are just trying to get a single question group
```

Comment:
This is to get around a limitation in Ember. I want to force the download of this question group, because I know it has changed in the backend. But if I simply do FLOW.store.find('QuestionGroup', id), it won't do it, because it already has that id in its local store. So I need to do a query, which are always done fresh.